lightning/src/routing/scoring.rs

```diff
@@ -2136,11 +2154,11 @@ mod tests {
 		};
 
 		let params = ProbabilisticScoringParameters {
-			liquidity_penalty_multiplier_msat: 40_000,
+			considered_impossible_penalty_msat: 42_420,
 			..ProbabilisticScoringParameters::zero_penalty()
 		};
 		let scorer = ProbabilisticScorer::new(params, &network_graph, &logger);
-		assert_eq!(scorer.channel_penalty_msat(42, &source, &target, usage), 80_000);
+		assert_eq!(scorer.channel_penalty_msat(42, &source, &target, usage), 42_420);
```

Comment:
Not sure I understand why this is the lower value. Also, does this test exercise the right code path even prior to this change? Seems like it won't hit the log approximation because `amount_msat` is `u64::max_value()`.